themes/digital.gov/layouts/landings/list.html

```diff
@@ -15,38 +15,25 @@ <h1>{{- .Params.title -}}</h1>
       </div>
       <h2 class="dg-landing__body-featured-resources">Featured Resources</h2>
       <ul class="usa-card-group" id="content-start">
-        {{- range .Pages -}}
+        {{- range .Pages }}
           <li class="usa-card tablet:grid-col-4">
-            <div class="usa-card__container">
-              <div class="usa-card__media usa-card__media--indent">
-                <div class="usa-card__img">
-                  {{- partial "core/img-flexible.html" (dict "Site" .Site "src" .Params.image "small" "true") -}}
-                </div>
-              </div>
-              <div class="usa-card__header">
-                <h2 class="usa-card__heading">
-                  {{- .Title | markdownify -}}
-                </h2>
-              </div>
-              <div class="usa-card__body">
-                <p>{{- .Params.summary | markdownify -}}</p>
-              </div>
-              <div class="usa-card__footer">
-                <a href="{{- .Permalink | absURL -}}" class="usa-button"
-                  >Explore {{ .Title | markdownify | lower }}</a
-                >
-              </div>
-            </div>
+            {{- $image := .Params.image -}}
+            {{- $layout := "no-image" -}}
+            {{- if $image -}}
+              {{- $layout = "has-image" -}}
+            {{- end -}}
+            {{- $cardData := dict "header" .Title "body" .Params.summary "link" (.Permalink | absURL) "link_title" (print "Explore " (.Title | markdownify | lower)) "layout" $layout "site" .Site -}}
```

Comment:
### Fix

Styles are broken for this button.

<img width="1680" alt="Screenshot 2024-08-05 at 3 38 12 PM" src="https://github.com/user-attachments/assets/82fd712f-61cc-4776-ab7a-335fbd7f8c4a">

Reply:
this is due to the class originally from the guides page applying universally in usa-card. Im introducing a new optional parameter that adds usa-button--outline if needed.